core-blocks/button/index.js

```diff
@@ -76,15 +72,8 @@ export const settings = {
 
 	attributes: blockAttributes,
 
-	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
-		const props = { 'data-resized': true };
```

Comment:
Handling for `const props = { 'data-resized': true };` shouldn't be removed in here.

Reply:
To be completely honest, I'm not sure about the purpose of the `const props = { 'data-resized': true };` here, I thought it was related to the old `align` attribute we are trying to get rid of. Could you enlighten me a bit regarding this if possible?

I've kept the specified line, however, I'm not sure if I've done it correctly. I'd be very grateful if you can review the change and let me know if it's correct. Thank you ❤️

Reply:
Me, too :) It might be something we had in the past which is no longer relevant.

Reply:
Got it 😊